compiler-core/src/error.rs

```diff
@@ -3365,6 +3368,14 @@ causing confusing errors and crashes.
                 location: None,
                 hint: Some("Rename this module and try again.".into()),
             }],
+
+            Error::ReplaceRequired => vec![Diagnostic {
+                title: "--replace flag not used to replace a release".into(),
+                text: "Attempted to publish an already published release without the --replace flag.".into(),
```

Comment:
Let's make this something like "There is already a version v1.0.0 of this library on Hex. This version has been recently published so you can replace it with new code, or you can publish using a different version number", with the wrap function.